javascript/src/config_checker.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
+    parseProperties: function (properties_json) {
+        let properties;
+        let properties_abs_path = path.resolve(properties_json);
+        try {
+            properties = require(properties_abs_path);
+        } catch (err) {
+            if (err instanceof SyntaxError) {
+                throw new Error(`${properties_json} does not follow JSON syntax`);
+            } else {
```

Comment:
Should this be a generic `else` or should we be looking for a specific type of FS Error? Should there be a separate message for "something went wrong, no idea what it was"? I can already see, e.g. file access denied not being accounted for in this logic.

Reply:
Okay we will add in the case where error is not identified then user gets the error message generated by machine.

Reply:
Fixed 7d274de83d26b96acb5de427115b208cfc7db26c